one_fm/hooks.py

```diff
@@ -72,7 +72,8 @@
 	"Purchase Order": "public/js/doctype_js/purchase_order.js",
 	"Journal Entry": "public/js/doctype_js/journal_entry.js",
 	"Payment Entry": "public/js/doctype_js/payment_entry.js",
-	"Item Price": "public/js/doctype_js/item_price.js"
+	"Item Price": "public/js/doctype_js/item_price.js",
+	"Employee Incentive": "public/js/doctype_js/employee_incentive.js"
```

Comment:
Just a Question: For any core doctype we need to add the custom path in hooks if it is not mentioned?

Reply:
Yes, or we can use export fixtures of client script